### DISCUSSION TITLE
SysVinitscript still not working (follow up of #1188

### DISCUSSION THREAD

**Discussion by kosh-mclien** [Kozea/Radicale#1413] [External Packaging]
I'm using Devuan 5 daedalus. (should be identical to Debian 12 bookworm).

I applied the patches from #1188 
I did however change the storage location in the config file:
`[storage]`
`...`
`filesystem_folder = <pathtomylocation>/radicale/collections`

As in #1188 manually starting works, init script does not. unfortunately I haven't been able to activate logging.

From my limited experience it seems to me, that the init-script is setting the path to the storagedir, too?
`CALDIR=/var/lib/$NAME`

**Comment by jonassmedegaard** [comment]
As mentioned also in #1188, please consider reporting issues with packaged code at the distributor of said packaged code.

Those packaging/redistributing the code has intimate knowledge of which issues are more likely caused by their changes and which are more likely "upstream" issues relevant to report here at the Github project.

Therefore: please consider report your issue at Devuan. Those maintaining the Deuan packaging of Radicale can then help assess if relevant to report same/similar issue to Debian, who (and that would be me) can help assess if relevant to report upstream here.

Sure, you are free to bypass distributors, but then you might experience that a) upstrem developers waste precious time investigating issues unreproducable to them, and b) issues fixed upstream but the fix not affecting your use of the code due to lack of coordination with your distribution vendoring of some code parts.

**Comment by kosh-mclien** [comment]
ok, I'm new to this bug reporting stuff.
But as I understand it, devuan uses the debian packages. So if I understand (at least) that part correctly. The debian maintainer (you?) would be the right person, but this is not the best place to file the bug report.
So that would be bugs.debian.org then?
EDIT: I think, I found it?
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1038930

How do I proceed now, as the patch is not working. A devuan bug report, I guess?

**Comment by jonassmedegaard** [comment]
One of the very differences between Debian and Devuan is how init-files are handled, so I urge you to contact your own distribution for how to report issues to them, instead of assuming that things behave the same across distributions.

Devuan developers will know when it is sensible to report issues in their distro upstream to Debian, so better to ask them than me about that.

Make sure to mention that you've already reported the issue here - to avoid duplicated reporting.

Thanks!

**Comment by kosh-mclien** [comment]
Just tried that, but there seems to be no maintainer for the Devuan radicale package, which means (if I get that correctly) that would be a direct downstream from debian. Or did I miss something here?
Sorry for being a noob-annoyance with this.

**Comment by jonassmedegaard** [comment]
Sorry, I cannot help you with how to interact with the distribution of your choice.  I can only warn you that if you bypass that, you risk bothered those you reach out to, because you potentially initiate an inefficient path of debugging.

Side note: Devuan is not my preferred distribution. Not because I am crazily in love with systemd (I am not), but because I am crazily in love with *maintained* code - which seems like a quality you can relate to right now in your current situation.

**Comment by kosh-mclien** [comment]
After understanding a bit more of the whole ways of bug tracking, I do understand, that this was addressed wrongly here (sorry for that). The problem itself is already reported at the right place:
[#1038930](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1038930) (existing patch, not implemented)
[#1055771](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1055771) (no patch so far)
